tach.toml

```diff
@@ -0,0 +1,83 @@
+exclude = []
```

Comment:
I guess it cannot read the dependencies from the `pyproject.toml`?

Reply:
I don't understand the comment. `exclude` is a list of project directories to exclude from checks.